common/_webdriver_qa_api/web/web_driver.py

```diff
@@ -69,3 +73,13 @@ def navigate_to(url: str):
 def close_cookie_consent():
     accept_button = get_webdriver_session().driver.find_element(by=By.XPATH, value="//button[@title='Accept']")
     accept_button.click()
+
+
+def open_new_tab():
```

Comment:
The method is duplicated - located in WebDriver class. Why is it a separate function?

Reply:
right, leave only separate function.
external function is more convenient to use from project layer